game.py

```diff
@@ -1,29 +1,35 @@
 import sys
 
+from Deck import Deck
 from GameState import GameState
+from Hand import Hand
 
 
 class Game:
     """
     The Game manages the control flow and solicits actions from agents.
     """
 
-    def __init__(self, landlordType, peasant1Type, peasant2Type, timeout):
+    def __init__(self, landlordType, peasant1Type, peasant2Type):
         # shuffle and deal deck
-        deck = None  # TODO UPDATE with deck object and .shuffle() with a random seed
+        deck = Deck()
 
         landlordCards = deck.deal(numCards=20)
         peasant1Cards = deck.deal(numCards=17)
         peasant2Cards = deck.deal(numCards=17)
 
         # create players with dealt hands of cards
-        landLordAgent = loadAgent(landlordType, landlordCards)
-        peasant1Agent = loadAgent(peasant1Type, peasant1Cards)
-        peasant2Agent = loadAgent(peasant2Type, peasant2Cards)
+        landLord = loadPlayer(landlordType, landlordCards, "LANDLORD")
+        peasant1 = loadPlayer(peasant1Type, peasant1Cards, "PEASANT")
+        peasant2 = loadPlayer(peasant2Type, peasant2Cards, "PEASANT")
```

Comment:
**Not important but worth thinking about**
To make the game more generic and fulfill future stretch goals, we might want to declare identical `Player` instances and assign their roles as part of the game (Since we might want `Player` to choose if they want to be the `LANDLORD`). I suggest having a random `Player` to be `LANDLORD` and assigning agents accordingly, but what you have here also works.